src/components/Modal/content/Send.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import styled from '@emotion/styled';
+
+import { dispatch } from 'store/rematch';
+import { selectChosenAsset } from 'store/selectors';
+
+import SendForm from 'components/Dashboard/widgets/Wallet/SendForm';
+import TxConfirmation from 'components/Dashboard/widgets/Wallet/TxConfirmation';
+
+const SendRoot = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Send = () => {
+  const [confirmation, setConfirmation] = useState(false);
+  const [recepient, setRecepient] = useState(null);
+  const [amountToSend, setAmountToSend] = useState(null);
+  const chosenAsset = useSelector(selectChosenAsset);
+
+  const handleSubmit = (address, amount) => {
+    dispatch.wallet.SET_CURRENT_TX_ID(null);
```

Comment:
Not sure if this is the best place to null the state before the tx starts 🤷🏽‍♀️  thoughts?

Reply:
Would this always be run before a `wallet.spend`? Because if so probably makes more sense to just include it in the `spend` effect.